tests/python/train/test_bucketing.py

```diff
@@ -20,9 +20,32 @@
 import mxnet as mx
 import random
 from random import randint
+from mxnet.contrib.amp import amp
 
 
-def test_bucket_module():
+def prepare_bucketing_data(buckets, len_vocab, batch_size, invalid_label, num_sentence):
+    train_sent = []
+    val_sent = []
+
+    for _ in range(num_sentence):
+        len_sentence = randint(6, max(buckets)-1) # leave out the two last buckets empty
```

Comment:
Is this comment correct? Randint doc says:
```
randint(a, b) method of random.Random instance
    Return random integer in range [a, b], including both end points.
```
So b is included in the range. Or am I missing something?